SK.WebApp/src/services/endpoints.ts

```diff
@@ -0,0 +1,13 @@
+export const apiAddress = 'https://localhost:44324';
```

Comment:
Please move this file to `src/endpoint.ts`

Reply:
done